src/libmongoc/src/mongoc/mongoc-write-command.c

```diff
@@ -1498,6 +1509,13 @@ _mongoc_write_result_complete (
       }
    }
 
+   /* If there is a raw error response then we know a server error has occurred.
+    * We should add the raw result to the reply. */
+   if (bson && !bson_empty (&result->raw_error_response)) {
+      bson_append_document (
+         bson, "serverResponse", -1, &result->raw_error_response);
```

Comment:
```suggestion
         bson, "errorReplies", -1, &result->raw_error_response);
```

Suggest naming "error" since this is only set when a server error occurs. And suggest using plural (along with other suggestion to store an array of server replies).